frontend/src/scss/global.scss

```diff
@@ -5,45 +5,72 @@ a {
     text-decoration: none;
 }
 
-#title {
-    font-family: $font-header;
-    font-size: 42px;
-    position: relative;
-    padding-bottom: 3rem;
+// Container classes
+
+.blue-container,
+.white-container {
+    background: $blue;
+    padding: 2rem 6rem;
 }
+.white-container {
+    background: $white;
+}
+
+// Container title classes
 
-#white-title {
+.white-title,
+.blue-title {
     font-family: $font-header;
-    font-size: 42px;
-    position: relative;
-    padding-bottom: 3rem;
+    font-size: 2.8rem;
+    color: $white !important;
+    display: flex;
+    justify-content: center;
+    padding: 3rem;
+
+    &::before {
+        content: '';
+        position: absolute;
+        bottom: 0;
+        left: 50%;
+        width: 250px;
+        height: 5px;
+        background-color: $white;
```

Comment:
ditto - You define `background-color` here and then you have to override it for `blue-title` which doesn't make any sense.